DependencyInjection/Traits/UsesQueueSerializer.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Smartbox\Integration\FrameworkBundle\DependencyInjection\Traits;
+
+use Smartbox\Integration\FrameworkBundle\Core\Serializers\QueueSerializerInterface;
```

Comment:
What's the purpose of this `use` statement here? It won't force you to implement what the interface defines.

Reply:
In this case is forcing to set a serializer that implements that interface

Reply:
Nvm i didnt see the setter type hinting